Forta-Bot-Challenge-4/COMP01/src/agent.ts

```diff
@@ -0,0 +1,51 @@
+import { Finding, HandleTransaction, TransactionEvent, FindingSeverity, FindingType } from "forta-agent";
+import { SUPPLY_EVENT_SIGNATURE } from "./constants";
+
+import { isOverThreshold } from "./utils";
+
+export function provideHandleTransaction(userAddress: String): HandleTransaction {
```

Comment:
While the README says that this should monitor a specific user, the `userAddress` is not actually used anywhere in the bot logic apart from the Finding metadata.